soccer/gameplay/GameplayModule.cpp

```diff
@@ -8,22 +8,42 @@
 
 #include <stdio.h>
 #include <iostream>
+#include <cmath>
 
 // for python stuff
 #include "robocup-py.hpp"
 
+using namespace Gameplay;
+
+REGISTER_CONFIGURABLE(GameplayModule);
+
 using namespace std;
 using namespace boost;
 using namespace boost::python;
 
 using namespace Geometry2d;
 
+ConfigDouble* GameplayModule::inset;
+
+void GameplayModule::createConfiguration(Configuration* cfg) {
+    inset = new ConfigDouble(cfg, "Field Edge Obstacle", .3);
+}
+
+bool GameplayModule::hasInsetChanged() {
+    if (abs(inset->value() - oldInset) > numeric_limits<double>::epsilon()) {
+        oldInset = inset->value();
+        return true;
+    }
+    return false;
+}
+
 Gameplay::GameplayModule::GameplayModule(SystemState* state)
     : _mutex(QMutex::Recursive) {
     _state = state;
 
     calculateFieldObstacles();
 
+    oldInset = .3;
```

Comment:
It would probably be better to set it equal to the config value here rather than a constant.